web3/contract/utils.py

```diff
@@ -356,6 +360,10 @@ def find_functions_by_identifier(
 def get_function_by_identifier(
     fns: Sequence[TContractFn], identifier: str
 ) -> TContractFn:
+    """
+    Check that the provided list of TContractFunction instances contains one element and
+    return it.
+    """
     if len(fns) > 1:
         raise Web3ValueError(
             f"Found multiple functions with matching {identifier}. " f"Found: {fns!r}"
```

Comment:
I see now we already phrase it this way with functions... take or leave really. Tiniest of nits.

```suggestion
            f"Found multiple functions with matching identifier `{identifier}`. Found: {fns!r}"
```